platform/ui/src/components/FilterWrapper/FilterWrapper.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '@ohif/ui';
+
+const baseLabelClassName =
+  'flex flex-col flex-1 text-white text-lg pl-1 select-none';
+const spanClassName = 'flex flex-row items-center cursor-pointer';
+const sortIconMap = {
+  '-1': 'sorting-active-down',
+  0: 'sorting',
+  1: 'sorting-active-up',
+};
+
+const FilterWrapper = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
+  onLabelClick,
+  className,
+  children,
+}) => {
+  const iconProps = {
+    name: isBeingSorted ? sortIconMap[sortDirection] : 'sorting',
+    className: classnames(
+      'mx-2 w-2',
```

Comment:
In general, the "outer element" of a React Component should not have margin.
The consumer of the component can set that, and this limits the places where this component can be used (with limited benefit)

In Viewer "view", this would be set on the outside of each element, and that would be cool beans

Reply:
This is actually the iconProps.className that is used only inside the "FilterWrapper".  
I agree with you and I think we are fine on this one.